apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -733,160 +759,169 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
     }
 
     /**
-    * @dev Internal function to check if a vote can receive delegated votes
-    *      It assumes the pointer to the vote is valid
-    * @return True if the given vote can receive delegated votes
+    * @dev Tell if a vote currently allows representatives to vote for delegated voters
+    * @param _vote Vote instance being queried
+    * @return True if the vote currently allows representatives to vote
     */
     function _canDelegateVote(Vote storage _vote) internal view returns (bool) {
         return _isActive(_vote) && !_withinOverruleWindow(_vote);
     }
 
     /**
-    * @dev Internal function to check if a voter can participate in a vote
-    *      It assumes the pointer to the vote is valid
-    * @return True if the given voter can participate a certain vote
+    * @dev Tell if a voter can participate in a vote.
+    *      Note that a voter cannot change their vote once cast, except for the principal voter
```

Comment:
I've stuck with the following terms for voters:

- Principal, delegated voter: always refer to the original token holder
- Representative: a delegate voting for others

Reply:
I think principal and representative are okay :)